conf/application-json-logger.xml

```diff
@@ -5,11 +5,11 @@
         <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
     </appender>
 
-    <logger name="application" level="WARN"/>
+    <logger name="application" level="INFO"/>
     <logger name="destinations" level="WARN"/>
     <logger name="uk.gov" level="WARN"/>
 
-    <root level="WARN">
+    <root level="INFO">
```

Comment:
I don't think it is a good idea to raise the log level to INFO in root.
I found for example that we log JWTPayload in info.
case class JwtPayload(
  sub: String,
  username: String,
  exp: Long,
  iss: String
)
don't you want to enable the info only for a particular package?